controllers/managedocs_controller.go

```diff
@@ -1520,6 +1628,67 @@ func (r *ManagedOCSReconciler) reconcileEgressNetworkPolicy() error {
 	return nil
 }
 
+func (r *ManagedOCSReconciler) reconcileEgressFirewall() error {
+	if r.DeploymentType != convergedDeploymentType {
```

Comment:
- a bit confused here, so this reconcile phase only happens for converged mode?
- @renan-campos can you help here? why are we skipping the egresspolicy for provider & consumer mode in https://github.com/red-hat-storage/ocs-osd-deployer/blob/0f906965804ef8d4bfa5f2de463d28bb95e6cf30/controllers/managedocs_controller.go#L1464-L1468 was this due for VPC patch?

Reply:
@leelavg in provider/consumer mode, the ceph pods are running in the host network namespace, so the ingress/egress network policies need to allow traffic between pods on the node network. However we can't set this without knowing what the CIDR range is for this network is first. By default it is 10.0.0.0/16, but it could be whatever it was initially set as. As a quick fix, the policies were disabled. But we do eventually need it to satisfy security requirements. This [PR](https://github.com/red-hat-storage/ocs-osd-deployer/pull/180) handles this, we just need it merged :pray:

Reply:
ack